ChromecastSampleApp/ChromecastSampleApp/Players/PlayerViewController.m

```diff
@@ -87,7 +87,9 @@ - (void)viewDidLoad {
   // Init the castManager in the ooyalaPlayer
   [self.ooyalaPlayer initCastManager:self.castManager];
   [self.ooyalaPlayer setEmbedCode:self.embedCode];
-  [self.ooyalaPlayer play];
+  if( self.mediaInfo.startPlaying ) {
```

Comment:
hmm at this point, we can get a castPlayer(), get it's embed code, and compare it to this embed code.  If it's the same, we can ignore the ooyalaPlayer.play.  how does that sound?

I'm not the biggest fan of modifying the mediaInfo